bundles/org.openhab.binding.omnikinverter/src/test/java/org/openhab/binding/omnikinverter/internal/test/OmnikInverterMessageTest.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.omnikinverter.internal.test;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openhab.binding.omnikinverter.OmnikInverterMessage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class OmnikInverterMessageTest {
+    private final Logger logger = LoggerFactory.getLogger(OmnikInverterMessageTest.class);
+
+    OmnikInverterMessage message;
+
+    @Before
+    public void setUp() {
+        byte[] b = new byte[1024];
+
+        File resourcesDirectory = new File("src/test/resources/omnik.output");
+        try (FileInputStream fis = new FileInputStream(resourcesDirectory);) {
+            fis.read(b);
+        } catch (IOException e) {
+            logger.debug("Could not read omnik.output test file" + e.getMessage());
+        }
+
+        message = new OmnikInverterMessage(b);
```

Comment:
You can probably also use `Files.readAllBytes` so you don't need to create a stream, e.g.:

```java
import java.io.File;
import java.nio.file.Files;


    @Before
    public void setUp() throws IOException {
        File file = new File("src/test/resources/omnik.output");
        message = new OmnikInverterMessage(Files.readAllBytes(file.toPath()));
    }

```